src/app/pages/access-code/access-code.page.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable eqeqeq */
```

Comment:
Given that you don't need to disable === enforcement here, can you remove this line?

Reply:
I will remove it